database/migrations/2022_04_26_204417_rename_author_to_author_id_news_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
```

Comment:
You're creating a fix migration to migration that wasn't even committed to main? Oh, come on.

Reply:
@krzysztofrewak I think he created that because per @kamilpiech97 request https://github.com/blumilksoftware/meetup/pull/39#discussion_r858268182